resources/public/api/conf/2.0/docs/match-endpoint-description.md

```diff
@@ -25,4 +25,5 @@
       <td><p>All characters must be identical</p></td>
     </tr>
   </tbody>
-</table>
\ No newline at end of file
+</table>
+<p>You will get a subset of the JSON response shown below based on your assigned scopes.</p>
```

Comment:
I don't _think_ this is correct - for this endpoint, successful matches will return the same response (with appropriate IDs etc filled in). I _believe_ HATEOAS links to the APIs are only served up from the other endpoint.

Reply:
That is a valid point. I'll correct